### ISSUE TITLE
Elección de temas para proyecto

### ISSUE THREAD

**Issue by UriAceves** [lbenet/2017-1_TSFisComp#78]
Temas para elegir:

 - Teorema de Sarkovski: ejemplos (y ejemplos en los que "no funciona" porque no se cumple alguna hipótesis).
- Billar abierto con tres obstáculos circulares en un triángulo equilátero: órbitas periódicas, dinámica simbólica, caos topológico, etc.
- Billar circular con un agujero: dinámica, tiempos de decaimiento.
- Ejemplo de dispersión caótica en un mapeo 2d: evolución de la silla caótica en términos del parámetros, bifurcaciones, hiperbolicidad.
- Mapeo estándard circular, número de rotación, lenguas de Arnold, etc.
- Efecto del ruido en sistemas dinámicos deterministas.

---
Temas elegidos:
* @cutsof, @Pablorv y @danmarurr 
    * Modelos discretos de control de poblaciones de insectos al introducir insectos estériles: estados estacionarios, diagrama de bifurcaciones, S>S_c tal que la población se elimina.
* @cpierard, @Oscarhg42 y @ferjefte 
    * Exponentes de Feigenbaum en el mapeo de Hénon: caso conservativo y caso disipativo.
* @miguelraz, @rodoart y @CarolinaLMH 
    * Modelos depredador-presa discretos: puntos de equilibrio.
---
Les pongo una lista con los temas. Para elegir tema deberán poner un comentario diciendo los miembros del equipo y el tema que desean. La lista se irá actualizando conforme vayan eligiendo y no se pueden repetir temas.

@Oscarhg42 @miguelraz @cpierard @danmarurr @CarolinaLMH @Yuriyama @rodoart @Pablorv @ferjefte @cutsof @Nogarx

cc: @lbenet

**Comment by danmarurr** [comment]
@cutsof, @Pablorv y yo elegimos el proyecto: 

"Modelos discretos de control de poblaciones de insectos al introducir insectos estériles: estados estacionarios, diagrama de bifurcaciones, S>S_c tal que la población se elimina."

**Comment by rodoart** [comment]
Saludos:
Me ha sido imposible asistir a la última clase.

Deseo el tema de modelos de predador-presa discretos.
Muchas gracias.

Rodolfo

**Comment by UriAceves** [comment]
Necesitas equipo para esto, si alguien más está interesado en tu proyecto puede comentarlo debajo y les asigno el tema.

**Comment by Yuriyama** [comment]
Vale, pero una cosa, ¿puedo elegir tema aunque aún no tenga derecho a proyecto?

Yuriko

**Comment by UriAceves** [comment]
Sí. Si resulta que en la fecha de corte acabas sin derecho será un asunto diferente.

**Comment by Yuriyama** [comment]
Ok, entonces me gustaría el teorema de Sarkovski. 

Yuriko

**Comment by UriAceves** [comment]
Vale, espera a ver si alguien quiere el mismo tema para que hagan equipo.

**Comment by cpierard** [comment]
Hola! @Oscarhg42 @ferjefte y yo queremos el tema: exponentes de Feigenbaum en el mapeo de Hénon: caso conservativo y caso disipativo.

**Comment by UriAceves** [comment]
Listo, ya lo tienen asignado.

[Errores en MC](http://www.northeastern.edu/afeiguin/phys5870/phys5870/node71.html)

[Henón Conservativo](https://books.google.com.mx/books?id=bDZze7YX64AC&pg=PA87&lpg=PA87&dq=conservative+henon+mapping&source=bl&ots=INvI8MqdQ1&sig=sbBrWma1HROO3gqBlCjocE4zncg&hl=en&sa=X&ved=0ahUKEwilpZeK6KvQAhVB92MKHfX8B-gQ6AEIKTAD#v=onepage&q=conservative%20henon%20mapping&f=false)

**Comment by miguelraz** [comment]
@rodoart:
Estoy buscando hacer equipo con @CarolinaLMH 

Te gustaria hacer mancuerna?

Saludos.

**Comment by rodoart** [comment]
@miguelraz
Por supuesto, por el momento no puedo asistir a la clase, pero trabajo desde mi casa. Podemos ponernos de acuerdo y revisar la última semana.

**Comment by miguelraz** [comment]
@UriAceves:
@rodoart @CarolinaLMH y yo haremos el tema de depredadores/presa discretos.

**Comment by UriAceves** [comment]
Listo, tienen el tema asignado.
